app/src/main/java/com/nmakademija/nmaakademija/fragment/ScheduleFragment.java

```diff
@@ -85,8 +85,28 @@ private void setScheduleItems(List<ScheduleEvent> scheduleEvents) {
 
         Collections.sort(scheduleItems, new ScheduleItemComparator());
 
+        int i;
+        Date now = new Date();
+        for (i = 0; i < scheduleItems.size() && scheduleItems.get(i).getDate().before(now); ++i) ;
+
         ScheduleAdapter adapter = new ScheduleAdapter(getContext(), scheduleItems);
-        scheduleListView.setAdapter(adapter);
+        scheduleRecyclerView.setAdapter(adapter);
+        scheduleRecyclerView.setLayoutManager(new LinearLayoutManager(getContext(), LinearLayoutManager.VERTICAL, false));
```

Comment:
You can set `layoutManager` in xml layout file with `app:layoutManager` property